src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -180,6 +186,31 @@ private BaseLauncherSettings getLauncherSettings(Path launcherDirectory) throws
         return launcherSettings;
     }
 
+    private boolean isServerAvailable() {
+        logger.trace("Checking Jenkins availability...");
+        final int timeout = 3000; // in milliseconds
+        HttpURLConnection connection = null;
+        try {
+            final URL serverURL = Objects.requireNonNull(BundleUtils.getURI("terasology_jenkins")).toURL();
```

Comment:
I don't think this has to come from a Bundle - compare to [`DownloadUtils.java`](https://github.com/MovingBlocks/TerasologyLauncher/blob/028b096d9d9bfc269c45ff892a546b3a1ef3bb94/src/main/java/org/terasology/launcher/util/DownloadUtils.java#L61) - you can just use a constant value.

Reply:
Hmm... then should I turn `JENKINS_JOB_URL` public (inside `DownloadUtils.java`) or create another constant inside `LauncherInitTask`?

Reply:
See comment above. Move the method to DownloadUtils and make it public. Then just introduce another constant value and use that.